### PR TITLE
Bump apache commons-compress from 1.18 to 1.19

### PR DESCRIPTION
**Pull request by mslowiak**
Closes https://github.com/strongbox/strongbox/issues/1528

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -64,6 +64,7 @@
         <version.carlspring.npm.metadata>1.0-SNAPSHOT</version.carlspring.npm.metadata>
 
         <version.apache.commons-io>2.6</version.apache.commons-io>
+        <version.apache.commons-compress>1.19</version.apache.commons-compress>
```
Comment (carlspring) [inline pom.xml]: I just noticed this...
Would you mind changing all the version properties that have hyphens in them to dots? (Here and in the `strongbox-proto-parent`, as well as all the other strongbox projects where they are used?

Comment (mslowiak) [reply pom.xml]: What is the reason to do this? :)

Comment (carlspring) [reply pom.xml]: It's cosmetic and for consistency only... All the other version properties are using dots only and not a mixture of dots and hyphens.

Comment (mslowiak) [reply pom.xml]: Okay then, I will fix it todays evening

Comment (carlspring) [reply pom.xml]: Great! Thanks a lot! :)

Comment (mslowiak) [reply pom.xml]: @carlspring This PR has been updated.
For `strongbox-proto-parent` fix is in PR: https://github.com/strongbox/strongbox-proto-parent/pull/7

Comment (carlspring) [reply pom.xml]: Fantastic!

...And you've checked that the `strongbox` project doesn't reference any of these properties, right? (The dependencies should be coming via the `dependencyManagement` section and be versionless across all the projects that extend these parents, but you never know...

Comment (mslowiak) [reply pom.xml]: Yes, I have checked that :)

Comment (carlspring) [reply pom.xml]: Excellent! Thank you! :)

**Pull request merged**